### ISSUE TITLE
Utilize SwiftGen for localized string

### ISSUE THREAD

**Issue by skyline75489** [lhc70000/iina#801]
**System and IINA version:**

- macOS 10.12.5
- IINA 0.0.11

With the help of SwiftGen we can eliminate hard-coded NSLocalizedString call.

See https://github.com/SwiftGen/SwiftGen#localizablestrings

**Comment by skyline75489** [comment]
Hard-coded NSImage calls can also be eliminated.

**Comment by skyline75489** [comment]
I just found https://github.com/lhc70000/iina/blob/develop/iina/Utility.swift#L53. This kind of hacky thing is why we need SwiftGen 🐼

**Comment by saagarjha** [comment]
I agree that we have a lot of hard coded things that we should address, but I'm not sure we need something like SwiftGen. `#imageLiteral` and `enum`s can go a long way…

**Comment by skyline75489** [comment]
SwiftGen just automagically generates `enum`s. Nothing else. So there isn't too much dependency. And it saves time, like, a lot.
